packages/wrangler/src/update-check.ts

```diff
@@ -16,11 +16,13 @@ export async function printWranglerBanner(performUpdateCheck = true) {
 	}
 
 	logger.log(
-		text +
+		"\n" +
+			text +
 			"\n" +
 			(supportsColor.stdout
 				? chalk.hex("#FF8800")("-".repeat(text.length))
-				: "-".repeat(text.length))
+				: "-".repeat(text.length)) +
+			"\n"
```

Comment:
Is there a reason we're changing the general shape of the banner? This will affect every Wrangler command, I think?

Reply:
indeed it will. My perception was that, without the new extra space, the logo tends to be squished by all other logs, which gives it all a crammed feeling. Like this, it gets some nicer spacing between it and everything else that gets logged after. But this is all purely my aesthetic feeling.

You can see what I'm referring to in the screenshot I added in the `wrangler logo display` section, for `wrangler dev`.